Build/Arch.pm

```diff
@@ -159,6 +159,13 @@ sub parse {
     my @assets = get_assets(\%vars, $asuf);
     push @{$ret->{'remoteassets'}}, @assets if @assets;
   }
+  my @exclarch;
+  push @exclarch, @{$vars{'arch'}};
+  @exclarch = grep {!/any/} @exclarch;
```

Comment:
this would also hit any possible arch containing "any" string. May not be a problem right now, but better write this as    {$_ ne "any"}    instead